### PR TITLE
#2674 Fixed reimbursement request bug

### LINKED ISSUE #2674: [Bug] - Denied request amounts are still included in the "currently owed" total
### Description

Reimbursemnt requests that are denied still have amounts showing in "currently owed"

### Acceptance Criteria

Currently owed only has the amounts for valid reimbursement requests.

### Proposed Solution

Filter out reimbursement requests to not include statuses with "DENIED".

### PR DESCRIPTION
**Pull request by AShyamkumar19**
## Changes

There was a bug in Finance. When a user adds a reimbursement request and later clicks on that request, they can click deny, and that cost will now be removed from the Currently Owed. 

## Screenshots

image

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] No `yarn.lock` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes #2674

### CODE REVIEW

**Review by Aaryan1203 — CHANGES_REQUESTED** [review]
prettier check is failing

**Review by Aaryan1203 — CHANGES_REQUESTED** [review]
Make sure to update your pr so its descriuptive

**Review by superhvarn — DISMISSED** [review]
LGTM

File: src/frontend/src/pages/FinancePage/RefundsSection.tsx [hunk 1]
```diff
@@ -121,8 +121,9 @@ const Refunds = ({ userReimbursementRequests, allReimbursementRequests }: Refund
     return <LoadingIndicator />;
 
   const displayedReimbursements = allReimbursements && tabValue === 1 ? allReimbursements : userReimbursements;
-  const displayedReimbursementRequests =
-    allReimbursementRequests && tabValue === 1 ? allReimbursementRequests : userReimbursementRequests;
+  const displayedReimbursementRequests = (
+    allReimbursementRequests && tabValue === 1 ? allReimbursementRequests : userReimbursementRequests
+  ).filter((request: ReimbursementRequest) => request.reimbursementStatuses[0].type !== ReimbursementStatusType.DENIED);
```
Comment (walker-sean) [inline src/frontend/src/pages/FinancePage/RefundsSection.tsx]: This looks at the first status, we actually want to check the most recent (last status in the array)

**Review by walker-sean — CHANGES_REQUESTED** [review]

**Review by superhvarn — APPROVED** [review]
Logic for the filter looks good to me

**Review by walker-sean — COMMENTED** [review]
LGTM

**Review by walker-sean — APPROVED** [review]

**Pull request merged**